aat/src/aat/resources/features/F-099 SearchParty (GlobalSearch)/F-099.feature

```diff
@@ -11,11 +11,11 @@ Feature: F-099: Search Party (Global Search)
     And it is submitted to call the [Import definition file] operation of [CCD Definition Store]
     Then a positive response is received
     And the response has all other details as expected
-    And a call [fetch Complex Fields Ordering Schema to verify single case field values in columns SearchPartyName(AC1), SearchPartyEmailAddress(AC6), SearchPartyAddressLine1(AC8), SearchPartyPostCode(AC10) and SearchPartyDoB(AC12)] will get the expected response as in [S-099.1_Get_ComplexFieldsOrdering].
+    And a call [fetch Complex Fields Ordering Schema to verify single case field values in columns SearchPartyName(AC1), SearchPartyEmailAddress(AC6), SearchPartyAddressLine1(AC8), SearchPartyPostCode(AC10), SearchPartyDoB(AC12) and SearchPartyCollectionFieldName] will get the expected response as in [S-099.1_Get_ComplexFieldsOrdering].
```

Comment:
Please switch to using a more up to date test definition file as adding `SearchPartyCollectionFieldName` to existing config is unlikely to pass additional validation added in [RDM-13106](https://tools.hmcts.net/jira/browse/RDM-13106).  i.e. reset this check and add a new verify step to check  `SearchPartyCollectionFieldName` using a different case type from the updated def file.

Reply:
Fixed in [3522292](https://github.com/hmcts/ccd-definition-store-api/pull/1130/commits/3522292204d6d0bc99422bd9abf6c58bbe2cbf8f)